challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -1017,7 +1017,7 @@
         {
           "text": "Template strings were used",
           "testString":
-            "getUserInput => assert(getUserInput('index').match(/`<li \\s*class\\s*=\\s*(\"\\s*text-warning\\s*\"|'\\s*text-warning\\s*')\\s*>\\s*\\$\\s*\\{(\\s*\\w+\\s*|\\s*\\w+\\s*\\[\\s*[\\w]+\\s*\\]\\s*)\\}\\s*<\\s*\\/li\\s*>`/g), 'Template strings were used');"
+            "getUserInput => assert(getUserInput('index').match(/`.*`/g), 'Template strings were used');"
```

Comment:
Unfortunately it seems like the commented out section is causing this to pass. I like your simplified test much more than the last one, so how about changing the commented out section?

Reply:
I am not finding that to be the case. 

![image](https://user-images.githubusercontent.com/26827682/43396875-f1abfd8c-93fa-11e8-836d-7594367203f2.png)

unless camper manually adds backticks to the comment

![image](https://user-images.githubusercontent.com/26827682/43396905-13425bf8-93fb-11e8-8ffb-77c0444d55d4.png)